src/pages/commits/commitChart/CommitChart.stories.tsx

```diff
@@ -0,0 +1,92 @@
+import { CommitChart } from "pages/commits/commitChart/CommitChart";
+import {
+  groupTasksByColor,
+  GroupedResult,
+} from "pages/commits/commitChart/utils";
+import { FlexRowContainer } from "pages/commits/CommitsWrapper";
+
+export default {
+  title: "Commit Charts",
+  component: CommitChart,
+};
+
+export const AbsoluteChart = () => (
+  <FlexRowContainer>
+    {groupedTaskData.map((item) => (
+      <CommitChart
+        groupedTaskStats={item.stats}
+        total={item.total}
+        max={max}
+        chartType="absolute"
+      />
+    ))}
+  </FlexRowContainer>
+);
+
+export const PercentChart = () => (
+  <FlexRowContainer>
+    {groupedTaskData.map((item) => (
+      <CommitChart
+        groupedTaskStats={item.stats}
+        total={item.total}
+        max={-1}
```

Comment:
should pass max in see how it behaves with it.

Reply:
This max is -1 because chart type is percentage.